src/mus2midi.c

```diff
@@ -0,0 +1,206 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+/*
+ * Reference: https://github.com/fawtytoo/Mus2Midi
+ *
+ */
+
+#include <stddef.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "mus2midi.h"
+
+typedef unsigned short WORD;
+
+typedef struct {
+    char id[4];
+    WORD scoreLen;
+    WORD scoreStart;
+} __attribute__((packed)) HDR_MUS;
+
+typedef struct {
+    char id[4];
+    int length;
+    WORD type;
+    WORD ntracks;
+    WORD ticks;
+} __attribute__((packed)) HDR_MID;
+
+char magicMus[4] = {'M', 'U', 'S', 0x1a};
```

Comment:
Hide the internal symbols as possible as you can. That is, add `static` qualifers.